Library/Formula/etcd.rb

```diff
@@ -2,11 +2,13 @@
 
 class Etcd < Formula
   homepage "https://github.com/coreos/etcd"
-  url "https://github.com/coreos/etcd/archive/v0.4.6.tar.gz"
-  sha1 "80b405fd01527eea6668fde0186ca6b119c1185c"
+  url "https://github.com/coreos/etcd/archive/v2.0.0.tar.gz"
+  # TOTO. I generated this myself on my machine. Can't find it on github
```

Comment:
The bot will double check the SHA itself and flag it up if it generates a different one :wink: